backend/inspirehep/records/indexer/tasks.py

```diff
@@ -81,6 +89,23 @@ def index_record(self, uuid, record_version=None, force_delete=None):
                 attempts=self.max_retries,
             )
         raise self.retry(countdown=backoff, exc=e)
+    except (
+        DisconnectionError,
+        TimeoutError,
+        InvalidatePoolError,
+        OperationalError,
+        UnboundExecutionError,
+        ResourceClosedError,
+    ) as e:
+        backoff = 2 ** (self.request.retries + 1)
+        if self.max_retries < self.request.retries + 1:
+            LOGGER.exception(
+                "DB Connection error. Scheduled retry for record",
+                uuid=uuid,
+                record_version=record_version,
+                force_delete=force_delete,
+            )
+        raise self.retry(countdown=backoff, exc=e)
```

Comment:
why not https://docs.celeryproject.org/en/v4.2.2/userguide/tasks.html#automatic-retry-for-known-exceptions ? :)

Reply:
Task requirements ;) But actually it's good idea, we should also do this for StaleDataError. I will create task for this.